source/operating-a-service/incident-playbook.html.md

```diff
@@ -17,14 +17,23 @@ Self-organise to appoint:
 
 The incident leads should [triage the incident](/operating-a-service/how-to-categorise-technical-incidents.html) (P1, P2, P3).
 
-### 3. Create an incident Slack channel and inform the stakeholders (comms lead)
+### 3. Identify any other services involved (tech lead)
+After triaging the issue, the tech lead should identify:
+- any upstream services which could be contributing to the issue
+- any downstream services likely to be affected by the issue
+and raise incidents where needed according to the [dependent services contact details](https://docs.google.com/document/d/1E3sL-Om_NPHWHdYdLykVuiWux_6AmR5pn5KjIkQYHaI/edit#bookmark=id.djbosiwhbjjy). 
```

Comment:
```suggestion
and raise incidents where needed according to the [dependent services contact details](https://docs.google.com/document/d/1E3sL-Om_NPHWHdYdLykVuiWux_6AmR5pn5KjIkQYHaI/edit#bookmark=id.5wrpa7wuk1l2). 
```
I've added a new bookmark in the Google Doc which deep-links directly to the dependent services contacts bit.